### PR TITLE
Add coverity pipeline

### PR DESCRIPTION
**Pull request by asomsiko**

### CODE REVIEW

File: .ci/azure/coverity.yml [hunk 1]
```diff
@@ -0,0 +1,297 @@
+resources:
```
Comment (azhogov) [inline .ci/azure/coverity.yml:1]: IMHO not needed

File: .ci/azure/coverity.yml [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+  repositories:
+  - repository: openvino_contrib
+    type: github
+    endpoint: openvinotoolkit
+    name: openvinotoolkit/openvino_contrib
+
+  - repository: testdata
```
Comment (azhogov) [inline .ci/azure/coverity.yml:8]: for tests only

File: .ci/azure/coverity.yml [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+    displayName: 'Submit for analysis'
+
+  - script: ls -alR $(REPO_DIR)/bin/
+    displayName: 'List bin files'
+
+  - script: cmake -DCMAKE_INSTALL_PREFIX=$(INSTALL_DIR) -P cmake_install.cmake
+    workingDirectory: $(BUILD_DIR)
+    displayName: 'Install'
+
+  - task: CMake@1
```
Comment (azhogov) [inline .ci/azure/coverity.yml:163]: tests below are not needed

File: .ci/azure/coverity.yml [hunk 4]
```diff
@@ -0,0 +1,297 @@
...
+  - script: |
+      set -e
+      $(REPO_DIR)/install_build_dependencies.sh
+      # Move jdk into contrib
+      sudo apt --assume-yes install openjdk-11-jdk
+      # For opencv-python: python3-setuptools and pip upgrade
+      python3 -m pip install --upgrade pip
+      python3 -m pip install -r $(REPO_DIR)/inference-engine/ie_bridges/python/requirements.txt
+      python3 -m pip install -r $(REPO_DIR)/inference-engine/ie_bridges/python/wheel/requirements-dev.txt
+      # For running Python API tests
```
Comment (azhogov) [inline .ci/azure/coverity.yml:88]: Remove reqs for tests

File: .ci/azure/coverity.yml [hunk 5]
```diff
@@ -0,0 +1,297 @@
...
+
+  - script: |
+        wget https://scan.coverity.com/download/linux64 --post-data "token=$(COVERITY_TOKEN)&project=openvino" -O coverity_tool.tgz
+        tar xvf coverity_tool.tgz --directory $(Agent.ToolsDirectory)
+        rm coverity_tool.tgz
+    workingDirectory: $(TEMP)
+
+  - script: $(Agent.ToolsDirectory)/cov-analysis*/bin/cov-build --dir cov-int ninja
+    workingDirectory: $(BUILD_DIR)
+    displayName: 'Build Lin'
```
Comment (azhogov) [inline .ci/azure/coverity.yml:143]: add coverity

File: .ci/azure/coverity.yml (hunk 1 above)
Comment (asomsiko) [reply .ci/azure/coverity.yml:1]: testdata remove, contrib is under scan

**Review by azhogov — APPROVED** [review]

**Pull request merged**